Source/WebKit/UIProcess/Cocoa/WebProcessPoolCocoa.mm

```diff
@@ -1240,6 +1248,15 @@ void setLockdownModeEnabledGloballyForTesting(std::optional<bool> enabledForTest
 void WebProcessPool::screenPropertiesChanged()
 {
     auto screenProperties = WebCore::collectScreenProperties();
+    if (m_suppressHDR) {
+        for (auto& properties : screenProperties.screenDataMap.values()) {
+#if HAVE(SUPPORT_HDR_DISPLAY)
+            auto maxHeadroom = std::min(std::max(properties.maxEDRHeadroom * .5f, 1.5f), 2.f);
```

Comment:
Can we use `std::clamp`, and some named constants for the upper/lower bounds. I think it'd be worth a comment about where the values came from too.